packages/theme/__tests__/style.test.ts

```diff
@@ -1,10 +1,13 @@
-import { getStyle } from '../src/BoxBase'
+import breakpoints from '../src/breakpoints'
+import { getStyle } from '../src/style'
 
 describe('getStyle', () => {
-  it('returns empty style', () => {
-    expect(getStyle({ sx: undefined })).toEqual({})
-    expect(getStyle({ sx: null })).toEqual({})
-  })
+  it.each([{}, { sx: undefined }, { sx: null }, { foo: 'bar' }])(
+    'returns style for %p',
```

Comment:
Shouldn't it say `'returns empty style for %p'`?

Reply:
Updated d650058